### PR TITLE
CI over Windows

### LINKED ISSUE #180: Run CI on a Windows container
Try with something like
```
  ci-windows:
    name: >
      Run Windows-based checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
    runs-on: ${{matrix.os}}
    strategy:
      matrix:
        otp_vsn: [23.2]
        os: [windows-latest]
    steps:
    - uses: actions/checkout@v2
    - uses: gleam-lang/setup-erlang@v1.1.2
      with:
        otp-version: ${{matrix.otp_vsn}}
      id: install_erlang
    - run: wget https://s3.amazonaws.com/rebar3/rebar3 -OutFile rebar3
      shell: powershell
    - run: |
        & "${{steps.install_erlang.outputs.erlpath}}\bin\escript.exe" rebar3 dialyzer
    - run: |
        & "${{steps.install_erlang.outputs.erlpath}}\bin\escript.exe" rebar3 ct
    - run: |
        & "${{steps.install_erlang.outputs.erlpath}}\bin\escript.exe" rebar3 cover
    - run: |
        & "${{steps.install_erlang.outputs.erlpath}}\bin\escript.exe" rebar3 escriptize
    - run: |
        & "_build\default\bin\elvis_core"
```
in `ci.yml`.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
~Hopefully~ Closes #180.

### CODE REVIEW

File: test/elvis_SUITE.erl [hunk 1]
```diff
@@ -374,7 +374,12 @@ find_file_and_check_src(_Config) ->
     [] = elvis_file:find_files(Dirs, "doesnt_exist.erl"),
     [File] = elvis_file:find_files(Dirs, "small.erl"),
 
-    {<<"-module(small).\n">>, _} = elvis_file:src(File),
+    {<<"-module(small).", LineBreak/binary>>, _} = elvis_file:src(File),
```
Comment (paulo-ferraz-oliveira) [inline test/elvis_SUITE.erl:377]: 👀

**Review by elbrujohalcon — APPROVED** [review]

File: test/elvis_SUITE.erl [hunk 2]
```diff
@@ -374,7 +374,12 @@ find_file_and_check_src(_Config) ->
...
+    LineBreak
+        = case os:type() of
+            {unix, _} -> <<"\n">>;
+            {win32, _} -> <<"\r\n">>
+        end,
```
Comment (g-andrade) [inline test/elvis_SUITE.erl:382]: ^ This approach will not work as expected if someone clones the `elvis_core` repo with auto-conversion from `\n` to `\n\r\` disabled, but that should be a very rare occurrence (and people developing Erlang on Windows are already rare enough to start with)

File: test/elvis_SUITE.erl (hunk 1 above)
Comment (g-andrade) [reply test/elvis_SUITE.erl:377]: Nice catch!

File: test/elvis_SUITE.erl (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply test/elvis_SUITE.erl:382]: Yeah, I thought about that edge case, but was mostly interested in the CI. If it breaks for somebody we'll deal with it then.
